pkgs/by-name/ne/netbox_4_0/package.nix

```diff
@@ -0,0 +1,132 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  plugins ? _ps: [ ],
+  nixosTests,
+}:
+let
+  py = python3.override {
+    packageOverrides = _final: prev: { django = prev.django_5; };
+  };
+
+  extraBuildInputs = plugins py.pkgs;
+in
+py.pkgs.buildPythonApplication rec {
+  pname = "netbox";
+  version = "4.0.11";
+
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "netbox-community";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-0yEz7v5RL1+cqbGDyuyEsywFonJQfPdVIQdL0qLyc04=";
+  };
+
+  patches = [
+    ./custom-static-root.patch
+    # From https://github.com/netbox-community/netbox/pull/17620
```

Comment:
Can't we fetchpatch this?

Reply:
We can't `fetchpatch` this one, it has conflicts when applied to version 4.0